test/integration/quota/quota_test.go

```diff
@@ -64,41 +57,20 @@ const (
 // 	quota_test.go:115: Took 12.021640372s to scale up with quota
 func TestQuota(t *testing.T) {
 	// Set up a API server
-	h := &framework.APIServerHolder{Initialized: make(chan struct{})}
-	s := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		<-h.Initialized
-		h.M.GenericAPIServer.Handler.ServeHTTP(w, req)
-	}))
-
-	admissionCh := make(chan struct{})
-	defer close(admissionCh)
-	clientset := clientset.NewForConfigOrDie(&restclient.Config{QPS: -1, Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}})
-	config := &resourcequotaapi.Configuration{}
-	admissionControl, err := resourcequota.NewResourceQuota(config, 5)
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	internalInformers := informers.NewSharedInformerFactory(clientset, controller.NoResyncPeriodFunc())
-	qca := quotainstall.NewQuotaConfigurationForAdmission()
-
-	initializers := admission.PluginInitializers{
-		genericadmissioninitializer.New(clientset, internalInformers, nil, nil, admissionCh),
-		kubeapiserveradmission.NewPluginInitializer(nil, nil, qca),
-	}
-	initializers.Initialize(admissionControl)
-	if err := admission.ValidateInitialization(admissionControl); err != nil {
-		t.Fatalf("couldn't initialize resource quota: %v", err)
-	}
```

Comment:
no longer needed?

Reply:
Yes - when using the unified way of setting the apiserver, resource quota admission is just enabled there - so we only need to configure it correctly (and this particular test is using default configuration, the other two below are using  the custom one so we need to create that config file).